### ISSUE TITLE
[authoring] autocomplete for internal links are broken

### ISSUE THREAD

**Issue by jessmartin** [dxos/dxos#6595]
- expected behavior is that ctrl-space would open an auto-complete dialog with the list of all internal objects (or at least documents)
- should be triggered by `[[` for discoverability
- people often have ctrl-space mapped to other things (spotlight, raycast, alfred, for example).

see also #5152

**Comment by ZaymonFC** [comment]
Blocked on https://github.com/dxos/dxos/issues/5187. We need better search infrastructure first.

**Comment by wittjosiah** [comment]
Duplicate of #5152
